ansible/inventory

```diff
@@ -0,0 +1,3 @@
+[localhost] 
+
+127.0.0.1 ansible_connection=local
```

Comment:
Can you also create another inventory with your VM there?

Reply:
you mean use the VM as a server to install apps?

Reply:
Nope, use it as a target machine. Just like you used it for testing. Run `ansible-playbook` from the host while targeting the VM.